lib/node_modules/@stdlib/string/base/stickycase/docs/types/index.d.ts

```diff
@@ -0,0 +1,48 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/**
+ * Converts a string to "sticky caps" case.
+ *
+ * @param str - input string
+ * @param p - probability of capitalization (between 0 and 1)
+ * @returns sticky case string
+ *
+ * @example
+ * var str = stickycase( 'hello world' );
+ * // returns <string>
+ *
+ * @example
+ * var str = stickycase( 'hello world', 0.2 );
+ * // returns <string>
+ *
+ * @example
+ * var str = stickycase( 'hello world', 0.8 );
+ * // returns <string>
+ *
+ * @example
+ * var str = stickycase( 'hello world', '0.5' );
+ * // throws TypeError
```

Comment:
For base packages, we omit runtime type checking of arguments.

For base packages, we omit runtime type checking of arguments.

```suggestion
```